firmware/src/transceiver_timing.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ * transceiver_timing.h
+ * Copyright (C) 2015 Simon Newton
+ */
+
+#ifndef FIRMWARE_SRC_TRANSCEIVER_TIMING_H_
+#define FIRMWARE_SRC_TRANSCEIVER_TIMING_H_
+
+/**
+ * @addtogroup transceiver
+ * @{
+ * @file transceiver_timing.h
+ */
+
+// Common params
+// ----------------------------------------------------------------------------
+
+/**
+ * @brief The minimum break time the user can configure, in microseconds.
+ *
+ * DMX1990 was 88uS and later versions raised this to 92uS. We allow down to
```

Comment:
All these s's for seconds should be lowercase (the u should be a mu, but we'll let that slide)